toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -1042,6 +1049,16 @@ void CellArea::drawLevelCell(QPainter &p, int row, int col, bool isReference) {
   TXshCell cell = xsh->getCell(row, col);
   TXshCell prevCell;
 
+  // If the user change the option to start a sequence with 0 to 1 in-live, we
+  // delete all cell and frame 0 of the actual Xsheet
+  if (!Preferences::instance()->isSequenceCanStartWith0() &&
+      cell.getFrameId().getNumber() == 0) {
+    cell.getSimpleLevel()->eraseFrame(TFrameId(0));
```

Comment:
Thanks for the correct point, @shun-iwasawa . I'll try to check it out as soon as possible.

You are right. This software is bit huge, and make yet little change it can become really complex. :)